src/proto/robot_config.proto

```diff
@@ -18,7 +18,110 @@ package bowler.robot_config;
 
 option java_multiple_files = true;
 option java_package = "com.commonwealthrobotics.proto.robot_config";
+//next 2
+// a center of mass object
+message CenterOfMass{
+  Pose centroid = 1;
+  double mass_kg = 2;
+}
 
-// Next: 1
+// next 3
+// a file location of a provider of some depency to be injected
+message GitFile {
```

Comment:
Use `FileSpec` from `git_f_s.proto` instead of defining a new type here. See [here](https://github.com/CommonWealthRobotics/bowler-proto/blob/b11054084600b9e85a5ee5922d7cf06579752f78/src/proto/script_host.proto) for an example of how to import and use message types.

Reply:
ok